tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/InstallationDetailsTest.java

```diff
@@ -0,0 +1,186 @@
+		private String remoteText = "";
+
+		private Logger log = new Logger(this.getClass());
+
+		public ConfigPropertiesHelper(String localText) {
+			this.localText = localText;
+
+			URL url;
+
+			try {
+				// get URL content
+				url = new URL(urlStr);
+				URLConnection conn = url.openConnection();
+
+				// open the stream and put it into BufferedReader
+				BufferedReader br = new BufferedReader(new InputStreamReader(conn.getInputStream()));
+				// loading to array list because the file is not too long
+				StringBuilder b = new StringBuilder();
+				br.lines().forEach(line -> b.append(line + "\n"));
+				remoteText = b.toString();
+				br.close();
+
+				log.info("Config file laoded from URL: " + urlStr);
+				log.debug("Config file content: \n" + remoteText);
+
+			} catch (MalformedURLException e) {
+				log.error("Could not get content at URL: " + urlStr);
+				e.printStackTrace();
+			} catch (IOException e) {
+				log.error("IOException occured while downlaoding configuration file at URL: " + urlStr);
+				e.printStackTrace();
+			}
+
+		}
+
+		public boolean isRegexValid(String regex) {
+			return remoteText.matches("(?s).*" + regex + ".*") && localText.matches("(?s).*" + regex + ".*");
+		}
+
+	}
+
+	// substring {majorVersion} is replaced with current major version
+	private final String[] regexArray = {
+			"jboss\\.discovery\\.directory\\.url\\|devstudio\\|{majorVersion}.*devstudio-directory\\.xml",
+			"jboss\\.discovery\\.site\\.url\\|devstudio\\|{majorVersion}.*=https://devstudio\\.redhat\\.com/{majorVersion}.*/updates/",
+			"jboss\\.discovery\\.earlyaccess\\.site\\.url\\|devstudio\\|{majorVersion}.*=https://devstudio\\.redhat\\.com/{majorVersion}.*/updates/discovery\\.earlyaccess/",
+			"jboss\\.discovery\\.earlyaccess\\.list\\.url\\|devstudio\\|{majorVersion}.*=https://devstudio\\.redhat\\.com/{majorVersion}.*/updates/discovery\\.earlyaccess/.*/devstudio-earlyaccess\\.properties",
+			"jboss\\.central\\.webpage\\.url\\|devstudio\\|{majorVersion}.*=https://repository\\.jboss\\.org/nexus/service.*artifact/maven/redirect.*public-jboss.*org\\.jboss\\.tools\\.central.*jbosstools-central-webpage.*zip",
+			"jboss\\.discovery\\.site\\.integration-stack\\.url\\|devstudio\\|{majorVersion}.*=https://devstudio\\.redhat\\.com/{majorVersion}.*integration-stack/discovery/",
+			"jboss\\.discovery\\.earlyaccess\\.site\\.integration-stack\\.url\\|devstudio\\|{majorVersion}.*=https://devstudio\\.redhat\\.com/{majorVersion}.*/development/updates/integration-stack/discovery/earlyaccess/",
+			"jboss\\.discovery\\.site\\.integration-stack-sap\\.url\\|devstudio\\|{majorVersion}.*=https://devstudio\\.redhat\\.com/{majorVersion}.*/development/updates/integration-stack/extras/" };
+
+	public String getJbossToolsMajorVersion() {
+		return jbossToolsVersion.substring(0, jbossToolsVersion.indexOf('.'));
+	}
+
+	@Before
+	public void setup() {
+		log.info("Navigate to installation details");
+		new WorkbenchShell().maximize();
```

Comment:
reddeer 2 does this by default before test execution

Reply:
I need to force the WorkbenchShell to maximize because in case of smaller screens it is hidden and can not be focused/opened. Discussed.